source/apiVolontaria/location/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Country(models.Model):
+    """ Model for countries
+
+        iso_code corresponds to the ISO 3166 norm
+
+    """
+
+    iso_code = models.CharField(max_length=2, primary_key=True)
+    name = models.CharField(max_length=45, blank=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name_plural = "Countries"
+        ordering = ["name", "iso_code"]
+        unique_together = (('iso_code', 'name'),)
+
+
+class StateProvince(models.Model):
+    """ Model for states and provinces
+
+        iso_code corresponds to the ISO 3166 norm
+
+    """
+
+    iso_code = models.CharField(max_length=3, primary_key=True)
+    name = models.CharField(max_length=55, blank=False)
+    country = models.ForeignKey(Country, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return "%s, %s" % (self.name, self.country.name)
+
+    class Meta:
+        verbose_name_plural = "StateProvinces"
+        unique_together = (('iso_code', 'name'),)
+
+
+class Address(models.Model):
+    """Model to store addresses for accounts"""
```

Comment:
Not specifically for accounts, can be for all item who have an address :)

Reply:
Sorry, copy/paste fail